recipes/timezone-data/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set name = "timezone-data" %}
+{% set version = "2024a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://data.iana.org/time-zones/releases/tzdata{{ version }}.tar.gz
+  sha256: 0d0434459acbd2059a7a8da1f3304a84a86591f6ed69c6248fffa502b6edffe3
+
+build:
+  number: 0
+  script:
+    - wget https://data.iana.org/time-zones/releases/tzdata{{ version }}.tar.gz
+    - mkdir -p $PREFIX/share/zoneinfo
+    - tar -xzf tzdata{{ version }}.tar.gz -C $PREFIX/share/zoneinfo
+    # This index file for the Windows time zone mapping is required by applications
+    # and libraries using the Time Zone Database, so we include it in the package for convenience.
+    - wget https://raw.githubusercontent.com/unicode-org/cldr/master/common/supplemental/windowsZones.xml -O $PREFIX/share/zoneinfo/windowsZones.xml  # [win]
```

Comment:
Should this maybe use the version too instead of pulling from master?

Reply:
Yes, definitely. This is addressed by 9f34123aeee6430c7718f308ec87fb6234bbdff7.

Should we make this package `noarch` or have a specific one for Windows?

Reply:
IMHO I would make it noarch generic. I dont think the overhead of that extra file is worth multiple builds. Not sure if that's common practice though.

Reply:
This extra file is not too voluminous, so I think using `noarch` is acceptable.